tpm2_pytss/types.py

```diff
@@ -1083,6 +1091,25 @@ def symcipher_from_secret(
         pub = TPM2B_PUBLIC(publicArea=pa)
         return (priv, pub)
 
+    def to_pem(self, data, password=None):
+        """Encode the key as PEM encoded ASN.1.
+
+        Returns:
+            Returns the PEM encoding as bytes.
+        """
+        return self.sensitiveArea.to_pem(data, password)
+
+    def to_der(self, public: TPMT_PUBLIC):
+        """Encode the key as DER encoded ASN.1.
+
+        public(TPMT_PUBLIC): The corresponding public key.
+
+        Returns:
+            Returns the DER encoding as bytes.
+        """
+
+        return self.sensitiveArea.to_der(serialization.Encoding.DER, public)
+
```

Comment:
Just a note, perhaps to_ssh might be useful, and add to_der + to_ssh for TPMT_PUBLIC/TPM2B_PUBLIC in the future

Reply:
Ill do private to_ssh now, and I filed a bug for the public half: https://github.com/tpm2-software/tpm2-pytss/issues/267